config/default.yml

```
@@ -3422,6 +3422,11 @@ Style/OneLineConditional:
   VersionAdded: '0.9'
   VersionChanged: '0.38'
 
+Style/OpenSSLDigestConstant:
+  Description: "Don't use algorithm constants for OpenSSL::Digest."
```

Can you enclose code with backticks? And I think that using `Reference` to indicate the information source will help users to understand :-)

```suggestion
  Description: "Don't use algorithm constants for `OpenSSL::Digest`."
  Reference: 'Please specify the information source URL here'
```